requirements/base.in

```diff
@@ -2,7 +2,6 @@
 -c constraints.txt
 
 Django                                  # Web application framework
-# openedx-events 6.0.0 has load_all_signals()
-openedx-events>=6.0.0                   # Events API
+openedx-events>=7.1.0                   # consumer api and consume_events command is provided
```

Comment:
OEP 18 has us using the end-of-line comment to just describe the overall purpose of the dependency, with a preceding-line comment used to explain any constraint.
```suggestion
# openedx-events 7.1.0 introduces consumer api and consume_events command
openedx-events>=7.1.0                   # Events API
```